e2e/scenario_test.go

```diff
@@ -869,7 +877,7 @@ func Scenario_ubuntu2204gpuNoDriver(t *testing.T) {
 
 func Scenario_ubuntu2204ContainerdURL(t *testing.T) {
 	RunScenario(t, &Scenario{
-		Description: "tests that a node using the Ubuntu 2204 VHD with the ContainerdPackageURL override bootstraps with the provided URL and not the maifest contianerd version",
+		Description: "tests that a node using the Ubuntu 2204 VHD with the ContainerdPackageURL override bootstraps with the provided URL and not the manifest containerd version",
```

Comment:
i believe at this point we no longer use manifest file and only utilize components.json, right @Devinwong ?

Reply:
I looked at the code that installs containerc and the place I found it (which might be the wrong place) it uses manifest.json not components.json

Reply:
Aaah, I was looking at the self-contained install scripts not the current scripts.